pyomo/core/plugins/transform/deactivate_trivial_constraints.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: UTF-8 -*-
+"""Transformation to deactivate trivial constraints."""
+import textwrap
+from math import fabs
+
+from pyomo.core.base.constraint import Constraint
+from pyomo.core.kernel.component_set import ComponentSet
+from pyomo.core.kernel.numvalue import value
+from pyomo.core.plugins.transform.hierarchy import IsomorphicTransformation
+from pyomo.util.plugin import alias
+
+__author__ = "Qi Chen <https://github.com/qtothec>"
```

Comment:
I recommend omitting the __author__ data.  The commit record captures this information.

Reply:
Sounds good. I think I originally put these in with the thought that my colleagues here know that they can bug me about support for them.